### PR TITLE
add SLU to contributors

### PR DESCRIPTION
**Pull request by HadrienG**

### CODE REVIEW

File: nf-core-contributors.yaml [hunk 1]
```diff
@@ -36,8 +36,8 @@ contributors:
     - full_name: Centre for Genomic Regulation
       short_name: CRG
       description: >
-        The Centre for Genomic Regulation (CRG) is an international biomedical research institute of excellence, 
-        based in Barcelona, Spain, whose mission is to discover and advance knowledge for the 
+        The Centre for Genomic Regulation (CRG) is an international biomedical research institute of excellence,
```
Comment (HadrienG) [inline nf-core-contributors.yaml]: my editor trims trailing whitespace sorry about that

**Comment by ewels** [comment]
Hah, the SVG has quite a lot of whitespace around it so ends up being just a wee bit tiny 😆 

image

**Comment by HadrienG** [comment]
This is the official one provided by SLU, I swear 😅 
I'll try to trim the white space and add a description

**Comment by ewels** [comment]
Logo fixed and looking a little more sensible now:

image

**Pull request merged**